src/frontend/pages/landing-page/LandingPage.jsx

```diff
@@ -0,0 +1,138 @@
+import './landing-page.css'
+import React from 'react'
+import NavgationTop from '../../components/navigation/NavigationTop'
+import { Link } from 'react-router-dom'
+import {ecom_logo, landing_image, signup_icon, login_icon, canon_eos_6d, nikkor_200mm, dsmc2, redvolt_microv, fisheye, nikon_z9} from '../../assets/index.js'
+
+
+export default function LandingPage (){
```

Comment:
Integrate mockbee in your project, you'll get a category file there in the backend folder. You can add your images, category name & description there, so that you'll have less of HTML code here. Another suggestion would be using a HomePageCard component to keep the landingpage.jsx clean